### PR TITLE
[Feature][Core] Added rest-api for batch start and stop

### LINKED ISSUE #7511: [Feature][Core] Added rest-api for batch start and stop
### Search before asking

- [X] I had searched in the [feature](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22Feature%22) and found no similar feature requirement.

### Description

Added rest-api for batch start and stop .

https://seatunnel.apache.org/docs/2.3.7/seatunnel-engine/rest-api

### Usage Scenario

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhangshenghang**
### Purpose of this pull request

- resolve #7511 

### How was this patch tested?

 
Local and e2e

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by Hisoka-X — COMMENTED** [review]
cc @liugddx

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpPostCommandProcessor.java [hunk 1]
```diff
@@ -103,11 +112,41 @@ private SeaTunnelServer getSeaTunnelServer() {
...
             throws IllegalArgumentException {
         Map<String, String> requestParams = new HashMap<>();
         RestUtil.buildRequestParams(requestParams, uri);
         Config config = RestUtil.buildConfig(requestHandle(httpPostCommand), false);
+
+        JsonObject jsonObject = createJsonObject(config, requestParams);
+        this.prepareResponse(httpPostCommand, jsonObject);
+    }
+
+    private JsonObject createJsonObject(Config config, Map<String, String> requestParams) {
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpPostCommandProcessor.java]:
```suggestion
    private JsonObject submitJobInternal(Config config, Map<String, String> requestParams) {
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @zhangshenghang !

**Review by liugddx — APPROVED** [review]
+1

**Pull request merged**